README.md

```diff
@@ -1,32 +1,32 @@
 # pelion-edge-provisioner (pep)
 
-It provisions the gateway for Pelion Edge solution to successfully and securely communicate with cloud. This tool hosts REST APIs and mongo database which helps create new and also store gateway-identites. The tool is designed to run on the factory floor and it provisions the gateways in production mode.
+The Pelion Edge provisioning tool provisions the gateway, so Pelion Edge can successfully and securely communicate with the cloud. This tool hosts REST APIs and a mongo database, which helps create and store gateway identities. The tool is designed to run on the factory floor, and it provisions the gateways in production mode.
```

Comment:
What does it mean to provision the gateway?
What's a mongo database?
What are gateway identities?

Reply:
What does it mean to provision the gateway? - This tool helps deploy the configuration on the gateway so that PE software knows which cloud to communicate to and which certificate to use while establishing that connection. 
What's a mongo database? - Its a NoSQL database which is used to keep record of all the identities (aka gateway configurations).
What are gateway identities? - It's a JSON object which has all the configuration parameters required by PE software to communicate with the Pelion cloud.